.github/workflows/checkstyle.yml

```
@@ -0,0 +1,13 @@
+name: Call Checkstyle
+
+on:
+  pull_request:
+    branches: ["master", "develop"]
+    paths:
+      - '**/*.java'
+
+jobs:
+  style-check:
+    uses: ./PSMRI/.github/workflows/checkstyle.yml@main
```

actionlint (1.7.4)

reusable workflow call "./PSMRI/.github/workflows/checkstyle.yml@main" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)